fuseimg/utils/visualization/imaging_multi_plot.py

```diff
@@ -0,0 +1,99 @@
+from typing import List , Callable , Any
+import matplotlib.pyplot as plt
+import numpy as np
+from functools import partial
+from fuseimg.utils.typing.typed_element import TypedElement
+from matplotlib.collections import PatchCollection
+from matplotlib.patches import Polygon
+
+def show_multiple_images(plot_seg : Callable , imgs : List, base_resolution :int = 5, **args ):
```

Comment:
args to kwargs

Reply:
plot_seg is a must argument?

Reply:
return value type is missing.

Reply:
changed.
it's a must, I think the idea that you can replace how you draw the segmentation